ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json

```diff
@@ -206,23 +206,35 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
-    "CaseFieldID": "applicant1ProceedWithClaimSpec2v1",
+    "CaseFieldID": "applicantDefenceResponseDocument",
     "PageDisplayOrder": 1,
     "PageFieldDisplayOrder": 12,
+    "DisplayContext": "READONLY",
+    "PageID": "RespondentResponse",
+    "ShowSummaryChangeOption": "N",
+    "FieldShowCondition": "respondent1ClaimResponseTypeForSpec = \"DO_NOT_SHOW_IN_UI\"",
+    "RetainHiddenValue": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
+    "CaseFieldID": "applicant1ProceedWithClaimSpec2v1",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 13,
     "DisplayContext": "MANDATORY",
     "PageID": "RespondentResponse",
     "ShowSummaryChangeOption": "Y",
     "RetainHiddenValue": "Y",
     "FieldShowCondition": "claimantResponseScenarioFlag=\"TWO_V_ONE\"",
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/mid/set-applicant1-proceed-flag"
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/mid/set-applicant-route_flags"
```

Comment:
Change to set-applicant-route-flags from set-applicant-route_flags

Reply:
fixed